types/index.d.ts

```diff
@@ -0,0 +1,141 @@
+// TypeScript Version: 3.0
+
+import {Node} from 'unist'
+import stringifyEntities = require('stringify-entities')
```

Comment:
```suggestion
import * as stringifyEntities from 'stringify-entities'
```

Reply:
If we just prefer `from`, I think we can do like

```ts
import {StringifyEntitiesOptions} from 'stringify-entities'
```

Reply:
Even better! :+1: